feature_map.gemspec

```diff
@@ -1,10 +1,10 @@
 Gem::Specification.new do |spec|
   spec.name          = 'feature_map'
-  spec.version       = '1.0.0'
+  spec.version       = '1.1.0'
   spec.authors       = ['Beyond Finance']
   spec.email         = ['engineering@beyondfinance.com']
   spec.summary       = 'A gem to help identify and manage features within large Ruby and Rails applications'
-  spec.description   = 'A gem to help identify and manage features within large Ruby and Rails applications'
+  spec.description   = 'FeatureMap helps identify and manage features within large Ruby and Rails applications. This gem works best in large, usually monolithic code bases for applications that incorporate a wide range of features with various dependencies.'
```

Comment:
If the `summary` and `description` attributes are identical, the `gem build feature_map.gemspec` prints the following warning message:

```
WARNING:  description and summary are identical
```